s4e-web/.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 14.15.5
```

Comment:
There is already a .tool-versions in project root with nodejs version declared. I'd update it there.

Reply:
alright - thanks for pointing it out - this PR is still in 'disaster state'...
Should I watch out for any other npm / node version apart from `s4e-web/pom.xml` and that `.tool-versions` ?

Reply:
> Should I watch out for any other npm / node version apart from `s4e-web/pom.xml` and that `.tool-versions` ?

A search through files only results in three places with nodejs version:
![image](https://user-images.githubusercontent.com/583197/107513917-36533580-6ba9-11eb-84fb-03e7a2e463a6.png)
I believe npm version isn't specified anywhere, apart from README files, which could be also updated, but won't result in any failure.

Reply:
Ok, I've moved `.tool-versions` to the root folder and updated references to node / npm.